.eslintrc

```diff
@@ -0,0 +1,31 @@
+root: true
```

Comment:
This file is in YAML, therefore could you place `---` as the first line please?

Also, could you confirm your PR would not work as-is if you named that file `.eslintrc.yml`? I'd prefer to see the extension in the filename, but only if ESLint accepts it (the doc and internets are saying it doesn't, but a confirmation from you after testing would be nice :-) ).